components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```diff
@@ -430,13 +443,14 @@ def main(argv):
         # Start Job Processing Loop
         while True:
             try:
-                search_and_schedule_new_tasks(
-                    db_conn,
-                    db_cursor,
-                    clp_metadata_db_connection_config,
-                    clp_config.archive_output,
-                    existing_datasets,
-                )
+                if False == recieved_signal:
```

Comment:
```suggestion
                if not received_sigterm:
```
Seems like we use `if not` more than `if False ==` more often than not for python (though this file uses `False ==` a bunch).